tao_compiler/mlir/disc/IR/lhlo_disc_ops.td

```diff
@@ -295,5 +295,26 @@ def LHLO_ArgsMutationOp : LHLODISC_Op<"args_mutation", []> {
   );
 }
 
+def LHLODISC_InplaceScatterOp: LHLODISC_Op<"inplace_scatter", []> {
```

Comment:
do we need a new operator to define a  **INPLACE** op? Maybe we simple way is `if (op.operand[0] == op.operand[-1])` ?

Reply:
Yeah you are right. Fixed.